docs/documentation/cn/sql-reference/sql-statements/Data Definition/ALTER TABLE.md

```diff
@@ -127,6 +127,10 @@ under the License.
                 TINYINT/SMALLINT/INT/BIGINT 转换成 TINYINT/SMALLINT/INT/BIGINT/DOUBLE。
                 LARGEINT 转换成 DOUBLE
                 VARCHAR 支持修改最大长度
+                DATETIME 转换成 DATE
```

Comment:
Are `Datetime to Date` and `Date to Datetime` related to precision conversion? If so, mark out the conversion principles as well.

Reply:
Done